microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+---
+title: What's new in DevTools (Microsoft Edge 105)
+description: "Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. Improved reliability in Issues tool. And more."
```

Comment:
I'd compress all 5 items into here, keeping And More at end.  character limit 350 chars[confirm] inside the quotes.